crates/sui-transactional-test-runner/src/test_adapter.rs

```diff
@@ -1938,7 +1967,8 @@ impl SuiTestAdapter {
             .collect();
 
         // Use a stable sort before assigning fake ids, so test output remains stable.
-        might_need_fake_id.sort_by_key(|id| self.get_object_sorting_key(id));
+        might_need_fake_id
+            .sort_by_key(|id| self.get_object_sorting_key(id, &accumulator_sort_keys));
```

Comment:
It seems a bit unnecessary to me that we pass the accumulator_sort_keys to get_object_sorting_key jus to process the accumulator events separately from the objects. I think you could just sort all objects, and sort all accumulator events, then merge them?
Also, I wonder if it's that useful to display the accumulator account ID alias in the TxnSummary in the first place. I think what we care more about is seeing which account gets changed in what way. So it may even be cleaner to just show `accumulators_written` as a list of (address_alias, type, operation) directly in the output?